### PR TITLE
Added bulk insert option 

### PR DESCRIPTION
**Pull request by Alexenon**

### CODE REVIEW

**Comment by Alexenon** [comment]
After review don't merge, this isn't the full version!

**Review by vldzz — COMMENTED** [review]
Nice work

**Comment by Alexenon** [comment]
@vldzz 
**Add this branch locally and check if the bulk insert works for you**
1. The project should be located in `C:/Desktop`
2. Change the user from `asus` to your windows/linux user name
3. If doesn't work let me know!

**Review by vldzz — APPROVED** [review]

**Pull request merged**
